docs/README.md

```diff
@@ -292,6 +293,13 @@ Therefore, in order to customize the `product-list` configuration, you can simpl
 | ---------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
 | `width`    | `number`    | Product image width (in pixels).  | `96`    |
 
+### `product-reference` props
+
+| Prop name              | Type      | Description                                                                                                                                                                                                                                     | Default value |
+| ---------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `identifierLabel`    | `string`    | Label to insert before the main reference information   | no default value    |
+| `identifierOption`    | `string`    | You can pull reference information through the following options: product id (`ProductId`), SKU ID (`ProductSkuItemId`), product reference ID (`ProductReferenceId`) and SKU reference ID (`ProductSkuReferenceId`). | `ProductReferenceId`    |
```

Comment:
```suggestion
| Prop name               |  Type         | Description                                        | Default value   |
| :------------------: | :---------: | :---------------------------------: | :-------------: |
| `identifierLabel`    | `string`    | Text label to be displayed to the left of the product reference value.  | `undefined`    |
| `identifierOption`    | `enum`    | Desired product reference data i.e. product identifier to be displayed. Possible options are: `ProductId`, `ProductSkuItemId`, `ProductReferenceId`, and `ProductSkuReferenceId`. | `ProductReferenceId`    |
```

Reply:
@guerreirobeatriz fixed!

Could you review it, please?